### ISSUE TITLE
About ACDC DSC Metric

### ISSUE THREAD

**Issue by QikaiZhang** [Amshaker/unetr_plus_plus#68]
Hi, I would like to ask the authors, is the dsc metric with a value of 92.83 in the ACDC dataset in your paper a validation set or is it a test set metric for inference, because when I look at the source code, your test set is treated as a validation set?

**Comment by Amshaker** [comment]
Hi @QikaiZhang ,

Thanks for the questions.

The DSC metric for ACDC dataset (92.83%) is based on the testing set not the validation set. The 100 patients are divided into 70 training samples, 10 validation samples, and 20 test samples. We use the same data division of nnFormer paper. Please note that the testing validation samples are defined here: https://github.com/Amshaker/unetr_plus_plus/blob/bc6bed1eb2d2d735d5f44740958ef4ea862304a6/unetr_pp/training/network_training/unetr_pp_trainer_acdc.py#L401C1-L402C1

Let me know if you have any questions.

Best regards,
Abdelrahman.

**Comment by QikaiZhang** [comment]
Thank you for your careful answer, but I think there are still some confusing points, in this py script you gave, in this code file, the way you divide your data is 80 for training and 20 for validation, and your validation part overlaps with the test part of nnFormer, 100 data samples you used all for training and validation, so what are your test set? Or give the serial number of these test samples.

Looking forward to your answer.

**Comment by siqingzhao36-SIQINGSIQING** [comment]
> 感谢你细致的回答，但我觉得还有一些混淆点，比如你给的这个 py 脚本，这个代码文件里，你用数据的分割方式是 80 用于训练，20 用于验证，而你的验证部分和 nnFormer 的测试部分重叠，100 个数据样本你用来训练和验证， 那么，你的测试集是什么？或者给出这些测试样品的序列号。
> 
> 期待你的回答。

“Hello, do you know what’s going on?”
